app/controllers/photo-comments/destroy.js

```diff
@@ -1,15 +1,12 @@
 import DestroyController from 'amber-ui/controllers/application/destroy';
+import { action } from '@ember/object';
 
-export default DestroyController.extend({
-  successMessage: 'Fotoreactie verwijderd!',
-  actions: {
-    destroy() {
-      this.set('photoAlbum', this.model.photo.get('photoAlbum'));
-      this._super(...arguments);
-    },
-    onSuccess() {
-      this._super(...arguments);
-      this.transitionToRoute('photo-albums.photo-album.show', this.photoAlbum);
-    },
-  },
-});
+export default class PhotoCommentDestroyController extends DestroyController {
+  successMessage = 'Fotoreactie verwijderd!';
+  successTransitionTarget = 'photo-albums.photo-album.show';
+  @action
+  async destroyModel() {
```

Comment:
fixed

Reply:
Now it redirects to the photo album it seems? Would it not be better if it redirects to the picture that the comment is attached to instead of the album? Or is that not possible?

Reply:
oops.

Reply:
I think I've actually fixed it now :)
it is definitely possible. takes a bit more effort, because I didn't build any options for multiple models to be used in the transition via the modelSaveUtils (the photo route takes an album id and a photo id.)
In addition, I didn't read your comment correctly. Was a bit late in the evening when I tackled this comment.